master/pkg/syncx/mapx/mapx_test.go

```diff
@@ -28,6 +30,38 @@ func FuzzMap(f *testing.F) {
 			m.WithLock(func(m map[uint8]string) {
 				_ = maps.Keys(m)
 			})
+		case 5:
+			_ = m.Values()
+		case 6:
+			m.Clear()
 		}
 	})
 }
+
+func TestMapx(t *testing.T) {
+	tests := []struct {
+		k string
+		v string
+	}{
+		{"1234", "hi"},
+		{"1235", "hello"},
+		{"1236", "world"},
+	}
+	testMap := New[string, string]()
+	for _, tt := range tests {
+		testMap.Store(tt.k, tt.v)
+	}
+	assert.Equal(t, len(tests), testMap.Len())
+	value, _ := testMap.Load("1235")
+	assert.Equal(t, "hello", value)
+	testMap.Delete("1235")
+	expectedValueList := [...]string{"hi", "world"}
+	valueList := testMap.Values()
+	assert.Equal(t, 2, testMap.Len())
+	sort.Strings(valueList)
+	for i, v := range valueList {
+		assert.Equal(t, expectedValueList[i], v)
+	}
+	testMap.Clear()
+	assert.Equal(t, 0, testMap.Len())
```

Comment:
nit: adding some space to this test would help, e.g.,
```suggestion
	assert.Equal(t, 2, testMap.Len())

	sort.Strings(valueList)
	for i, v := range valueList {
		assert.Equal(t, expectedValueList[i], v)
	}
	
	testMap.Clear()
	assert.Equal(t, 0, testMap.Len())
```